lib/content-services/src/lib/category/categories-management/categories-management.component.html

```diff
@@ -1,10 +1,19 @@
 <div class="adf-categories-management">
-    <p *ngIf="!categories.length && !categoryNameControlVisible"
-       class="adf-no-categories-message">
-        {{ noCategoriesMsg | translate }}
-    </p>
-    <div class="adf-categories-list"
-         [class.adf-categories-list-fixed]="!categoryNameControlVisible">
+    <div *ngIf="((!categoryNameControlVisible && categories.length)) || categoryNameControlVisible"
```

Comment:
this logic becomes too long, please optimise via separate function, for example:

```ts
get isNameCategoryVisible(): boolean {
 return (!categoryNameControlVisible && categories.length)) || categoryNameControlVisible;
}
```

and then 

```html
<div *ngIf="isNameCategoryVisible">...</div>
```

ensure you have unit tests for this